ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -340,7 +340,7 @@ objects:
       - name: INSTALL_INITIAL_RELEASE
         value: ${INSTALL_INITIAL_RELEASE}
       - name: RELEASE_IMAGE_INITIAL
-        value: ${RELEASE_IMAGE_INITIAL}
+        value: default-route-openshift-image-registry.apps.build01.ci.devcluster.openshift.com/ci-op-626wnr1x/release:latest
```

Comment:
had to use diffrerent release image bacause of bug in openshift-sdn  that failing our installations.